src/kernel/include/shared/cos_types.h

```diff
@@ -223,40 +223,54 @@ captbl_idsize(cap_t c)
  * LLBooter initial captbl setup:
  * 0 = sret,
  * 1-3 = nil,
- * 4-5 = this captbl,
- * 6-7 = our pgtbl root,
- * 8-11 = our component,
- * 12-13 = vm pte for booter
- * 14-15 = untyped memory pgtbl root,
- * 16-17 = vm pte for physical memory,
- * 18-19 = km pte,
- * 20-21 = comp0 captbl,
- * 22-23 = comp0 pgtbl root,
- * 24-27 = comp0 component,
- * 28~(20+2*NCPU) = per core alpha thd
+ * 4-7 = memory fault handler,
+ * 8-11 = divide by zero fault handler,
+ * 12-15 = breapoint fault handler,
+ * 16-19 = invaild instruction fault handler,
+ * 20-23 = bound exceed fault handler,
+ * 24-27 = component not exists fault handler,
+ * 28-31 = sinv to invaild component fault handler,
+ * 32-33 = this captbl,
+ * 34-35 = our pgtbl root,
+ * 36-39 = our component,
+ * 40-41 = vm pte for booter
+ * 42-43 = untyped memory pgtbl root,
+ * 44-45 = vm pte for physical memory,
+ * 46-47 = km pte,
+ * 48-49 = comp0 captbl,
+ * 50-51 = comp0 pgtbl root,
+ * 52-55 = comp0 component,
+ * 56~(48+2*NCPU) = per core alpha thd
```

Comment:
This comment has fallen out of coherence with the code...

We need to add the other arrays as well.